pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/CreateSegmentCommand.java

```diff
@@ -413,6 +447,32 @@ public void run() {
     return executor.awaitTermination(1, TimeUnit.HOURS);
   }
 
+  private boolean verifySegment(File indexDir) {
+    File localTempDir = FileUtils.getTempDirectory();
```

Comment:
You cannot directly use the tempDir and delete it after the verification. Multiple segments will end up modifying and deleting the same tempDir.
You can use `
    new File(FileUtils.getTempDirectory(), org.apache.pinot.common.utils.FileUtils.getRandomFileName())` as the tempDir